site/content/docs/user/quick-start.md

```diff
@@ -179,9 +181,11 @@ the resulting node image using the flag `--image`.
 If you previously changed the name and tag of the base image, you can use here
 the flag `--base-image` to specify the name and tag you used.
 
-**Note**: If you are running kind on MacOS or Windows then it is recommended
-that you have at least 8GB of RAM dedicated to the virtual machine (VM) running
-the Docker engine otherwise Building Kubernetes may fail.
+
+### Settings for Docker Desktop
+
+If you are building using kind on MacOS or Windows then you need a minimum of 6GB of RAM 
```

Comment:
building [Kubernetes] ?
might be worth being explicit

Reply:
sure - put in an example command :)